src/pdn/test/power_switch.api.tcl

```diff
@@ -0,0 +1,52 @@
+source "helpers.tcl"
+
+read_lef ../../../test/sky130hd/sky130hd.tlef 
+read_lef ../../../test/sky130hd/sky130_fd_sc_hd_merged.lef 
+
+read_lef sky130hd/power_switch.lef 
+
+read_def power_switch/2_5_floorplan_tapcell.def
```

Comment:
to be consistent, please rename to sky130_power_switch/floorplan.def (name should make it clear the process)

Reply:
Done